### ISSUE TITLE
Fails to parse comments containing bracket characters

### ISSUE THREAD

**Issue by vinceplusplus** [Flinesoft/BartyCrouch#45]
If I have the following code:
```swift
let s1 = NSLocalizedString("test", comment: "test comment containing ()")
```

The generated .strings file becomes:
```
"test" = "";
```

There seems to be a bug in the comment matching pattern in the recent multiline comment support. I took the solution from http://blog.ostermiller.org/find-comment and adapted it a bit (removed newline characters thanks to .dotMatchesLineSeparators and added extra brackets to keep the third last capture group pointing to the comment), now it seems to work fine:
```swift
let translationRegexString = "(?:\\s*/\\*(((?:[^*]|(?:\\*+(?:[^*/])))*))\\*+/\\s*)?\\s*(?:^\\s*\"([\\S ]*)\"\\s*=\\s*\"([\\S ]*)\"\\s*;\\s*$)"
```

**Comment by Dschee** [comment]
Could you post this as a pull request so I can see if tests are passing and merge directly? Also if you could alter the tests so the brackets case is covered, that would be very helpful. Thanks for you help!

**Comment by vinceplusplus** [comment]
I've just created a pull request #46. I made some changes in project settings in order to run tests (changed some names), but I didn't include those changes in the pull request. My Xcode 8 complains:
```swift
@testable import BartyCrouch
```

**Comment by Dschee** [comment]
I've recognized that after running the CLI target one has to clean-build in order for the framework tests to work. Maybe that fixes it? CI seems to run fine, so I'm merging as is. I'd be glad if you could keep me updated about the complaint though. Thanks! 🎉

**Comment by Dschee** [comment]
Closed due to merge of #46.
